dot/parachain/candidate-validation/candidate_validation_test.go

```diff
@@ -22,9 +22,11 @@ var (
 	paramsTooLarge   = ParamsTooLarge
 	codeHashMismatch = CodeHashMismatch
 	badSignature     = BadSignature
+	invalidOutputs   = InvalidOutputs
+	badParent        = BadParent
 )
 
-func createTestCandidateReceiptAndValidationCode(t *testing.T) (
+func createTestCandidateReceiptAndValidationCodeWParaId(t *testing.T, id uint32) (
 	parachaintypes.CandidateReceipt, parachaintypes.ValidationCode) {
```

Comment:
make it helper function
```suggestion
func createTestCandidateReceiptAndValidationCodeWParaId(t *testing.T, id uint32) (
	parachaintypes.CandidateReceipt, parachaintypes.ValidationCode) {
	t.Helper()
```

Reply:
Updated.